tests/libdnf/transaction/RpmItemTest.cpp

```diff
@@ -112,7 +112,9 @@ RpmItemTest::testGetTransactionItems()
     std::chrono::duration< double > read_duration = read_finish - read_start;
 
     auto createMs = std::chrono::duration_cast< std::chrono::milliseconds >(create_duration);
+    (void)createMs;
     auto readMs = std::chrono::duration_cast< std::chrono::milliseconds >(read_duration);
+    (void)readMs;
```

Comment:
I thought about removing these ones but it would have unwound a lot of code and I assume the goal was to use this in the future, so suppressing via `(void)` is simplest.